core/src/main/java/org/apache/accumulo/core/metadata/schema/TabletMetadata.java

```diff
@@ -106,6 +107,7 @@ public class TabletMetadata {
   private OptionalLong compact = OptionalLong.empty();
   private Double splitRatio = null;
   private Map<ExternalCompactionId,ExternalCompactionMetadata> extCompactions;
+  public boolean chopped = false;
```

Comment:
Would be better to make this private and have a method to access it, or could not have it at all.  The reason a method would be good is that `ensureFetched()` could be called in  the method to make sure it was actually retrieved.  All other fields do this.  In avoids bugs where someone did not fetch a field from the metadata table but they try to use it.  Like if someone read this chopped field but never fetched it, then it would always be false.

Reply:
Added a getter and call to ensureChecked in [c2e5554](https://github.com/apache/accumulo/pull/2969/commits/c2e555499e06daa220dd5982a6a021f599dbdfe8)